src/main/java/fr/zcraft/quartzlib/components/i18n/translators/gettext/PluralForms.java

```diff
@@ -139,26 +185,41 @@ private String normalizeFormsScript(final String script) {
             // English, German, Dutch, Swedish, Danish, Norwegian, Faroese, Spanish, Portuguese, Italian,
             // Greek, Bulgarian, Finnish, Estonian, Hebrew, ahasa Indonesian, Esperanto, Hungarian, Turkish
             case "n!=1":
+            case "n<1||n>1": // Yup. POEdit can generate this.
```

Comment:
Is it sure that we can't get "n>1||n<1" ?

Reply:
That would be equivalent to `1` i.e. only using one form but always the 2nd one. That would be weird. Having two forms but always using the second one?

For languages with one form for both singular and plural, we use `0`.

Reply:
Never seen but we can add it just in case

Reply:
They seems to have a logic behind the order, so n>1||n<1, is unlickely